### PR TITLE
Fix read statement

### LINKED ISSUE #163: read statements are flagged as errors
`read (so70, knum=3, dir=0, end=*next)`
image
image
image

### PR DESCRIPTION
**Pull request by Lotes**
Closes #163

### CODE REVIEW

**Comment by Lotes** [comment]
We should wait until #196 is merged, because the right parenthesis is also causing issues.

**Review by dhuebner — APPROVED** [review]
We need to check the marked parameter like `knum`, there is something wrong... 

 
Bildschirmfoto 2024-11-07 um 15 50 06

**Pull request merged**
